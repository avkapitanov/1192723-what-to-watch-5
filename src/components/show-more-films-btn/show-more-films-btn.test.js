import React from "react";
import renderer from "react-test-renderer";

import ShowMoreFilmsBtn from "./show-more-films-btn";

it(`ShowMoreFilmsBtn component render correctly`, () => {
  const tree = renderer.create(
      <ShowMoreFilmsBtn
        onChangeFilmsCount={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
