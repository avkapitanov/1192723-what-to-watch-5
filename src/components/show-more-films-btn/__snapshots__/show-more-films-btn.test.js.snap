// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMoreFilmsBtn component render correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
